assets/styles/cookie-notice.scss

```diff
@@ -16,27 +16,22 @@
   &[hidden] {
     display: none;
   }
-}
 
-.cookie-notice__link {
-  &,
-  &:hover,
-  &:focus {
-    color: currentColor;
-    text-decoration: underline;
-  }
+  a {
+    &,
+    &:hover,
+    &:focus {
+      color: currentColor;
+      text-decoration: underline;
+    }
 
-  &:hover,
-  &:focus {
-    opacity: 0.8;
+    &:hover,
+    &:focus {
+      opacity: 0.8;
```

Comment:
Not sure why we are fiddling with the opacity here. Might also be a remnant of the former button styling that is unnecessary for a text link now.

Reply:
This styling is required and wasn't a leftover from the previous cookie notice.

It's to provide a hover state without relying on a particular text color. For example, if the color of the notice was changed to `#000` for a particular client site, there would still be a suitable hover state without having to specify another color.